Lib/test/test_cgi.py

```diff
@@ -353,6 +353,23 @@ def test_fieldstorage_part_content_length(self):
         self.assertEqual(fs.list[0].name, 'submit-name')
         self.assertEqual(fs.list[0].value, 'Larry')
 
+    def test_field_storage_multipart_no_content_length(self):
+        fp = BytesIO(b"""--MyBoundary
+Content-Disposition: form-data; name="my-arg"; filename="foo"
+
+Test
+
+--MyBoundary--
+""")
+        env = {
+            "REQUEST_METHOD": "POST",
+            "CONTENT_TYPE": "multipart/form-data; boundary=MyBoundary",
+            "wsgi.input": fp,
+        }
+        fields = cgi.FieldStorage(fp, environ=env)
+
+        assert len(fields["my-arg"].file.read()) == 5
```

Comment:
`unittest.TestCase` methods (like `assertEqual()`) should be used instead of raw `assert`s.

Reply:
Thanks, done in commit d67919b11e9cf0f6f98419938123ad6a38bfd5da